tools/shared.py

```diff
@@ -2737,7 +2758,8 @@ def path_to_system_js_libraries(library_name):
 
     elif library_name.endswith('.js') and os.path.isfile(path_from_root('src', 'library_' + library_name)):
       library_files += ['library_' + library_name]
-    else:
+    elif not Settings.WASM_BACKEND:
+      # The wasm backend will report these when wasm-ld runs
```

Comment:
maybe we could have an assert on them here?

Reply:
Can you explain?

The idea is that fastcomp errors out here on missing libraries.. but with the wasm-backend the error will happen later.

Reply:
Oh, later, not earlier, I misunderstood. sgtm.